src/Provider/WFSProvider.js

```diff
@@ -63,8 +62,8 @@ function getFeatures(crs, tile, layer) {
 
     const urld = URLBuilder.bbox(tile.extent.as(layer.crs), layer);
 
-    if (cache.has(urld)) {
-        return Promise.resolve({ feature: cache.get(urld) });
+    if (Cache.has(urld)) {
+        return Promise.resolve({ feature: Cache.get(urld) });
```

Comment:
You could use the same pattern as above.
```js
return Cache.get(urld) || Cache.set(urld, Fetcher.json(...).then(...)
```

Reply:
Done !

Reply:
@peppsac Your last comment was taken into account, we can merge?